### ISSUE TITLE
Get_followers non-valid next_cursor value

### ISSUE THREAD

**Issue by Facj** [mkearney/rtweet#48]
I am using get_followers function to get followers of a group of users in a dataset. I have found out that the output from the function does not always look the same. Sometimes the ouput is a dataframe with a ids column and a valid cursor.

> followers <- get_followers("33344949",n="all")
> str(followers)
> >'data.frame':	42012 obs. of  1 variable:
> >  $ ids: num  8.16e+17 7.83e+17 8.16e+17 8.09e+17 8.16e+17 ...
> >   attr(*, "next_cursor")= chr "0"
> >  NULL

> next_cursor(followers)
> >"0"

However, some other times the output is a 5 column dataframe,  where the cursor is not a character.  

> followers <- get_followers("110938362",n="all")
> str(followers)
> > 'data.frame':	882 obs. of  5 variables:
> >  $ ids: num  1.11e+09 5.54e+08 7.51e+07 8.07e+07 2.88e+08 ...
> >  $ NA : int  0 0 0 0 0 0 0 0 0 0 ...
> >  $ NA : chr  "0" "0" "0" "0" ...
> >  $ NA : int  0 0 0 0 0 0 0 0 0 0 ...
> >  $ NA : chr  "0" "0" "0" "0" ...
> >  attr(*, "next_cursor")=function (ids) 
> > NULL

> next_cursor(followers)
> > function (ids) 
> > {
> >     attr(ids, "next_cursor")
> > }
> > <environment: namespace:rtweet>
> 

Am I doing something wrong?

**Comment by mkearney** [comment]
I don't think so. I've been overhauling package so it's probably something internally. I'll give it a look in the next few hours!

**Comment by mkearney** [comment]
It worked fine for me, but it's likely due to something I recently fixed. Not too long ago I changed the output so the variable name would be "user_id" (to make consistent with rest of rtweet data). Try to update to the current github version (which will have the ids named "user_id" in addition to the fixes to the parsing process) and see if it's better. 

```{r}
devtools::install_github("mkearney/rtweet")
library(rtweet)
followers <- get_followers("33344949", n = "all")
followers %>% str()
followers <- get_followers("110938362", n = "all")
followers %>% str()
```
